app/src/main/java/fr/free/nrw/commons/upload/UploadActivity.java

```diff
@@ -469,7 +471,18 @@ public void onNextButtonClicked(int index) {
             vpUpload.setCurrentItem(index + 1, false);
             fragments.get(index + 1).onBecameVisible();
         } else {
-            presenter.handleSubmit();
+            if(defaultKvStore.getInt(COUNTER_OF_NO_LOCATION, 0) == 10){
```

Comment:
Would you mind changing `==10` to `>= 10`?
It is a usual pattern, for safety.

Reply:
Sure